feat: split trustLevel into separate options (#9388)
renovatebot/renovate: docs/usage/self-hosted-configuration.md

## endpoint

## exposeAllEnv

> By default, Renovate will only pass a limited set of environment variables to package managers.
> Potentially, there could be leaks of confidential data if a script you don't trust enumerates all values in env, so set this to true only if you trust the repositories which the bot runs against.

Reviewer: ```suggestion
By default, Renovate only passes a limited set of environment variables to package managers.

Confidential data can be leaked if a malicious script enumerates all environment variables.
Set `exposeAllEnv` to `true` only if you have reviewed (and trust) the repositories which Renovate bot runs against.
```

How about this?

> Setting this to true will also allow for variable substitution in `.npmrc` files.

Reviewer: ```suggestion
Setting this to `true` will also allow for variable substitution in `.npmrc` files.
```
